docker/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM nvidia/cuda:12.2.0-devel-ubuntu22.04
+LABEL maintainer="williamhilton.works@gmail.com"
+
+# Note: This Dockerfile is based on the Kaldi official Dockerfile. We needed to remove --use-cuda flag from the
```

Comment:
Then, does the Kaldi build actually link to CUDA?

Reply:
You can e.g. run `cuda-gpu-available`, which will check if CUDA was linked and if a GPU is available.